packages/dx-react-grid/src/plugins/table-inline-cell-editing.tsx

```diff
@@ -18,115 +21,131 @@ const pluginDependencies = [
   { name: 'DataTypeProvider', optional: true },
 ];
 
+const rowsWithEditingCellsComputed = (
+  { tableBodyRows, editingCells }: Getters,
+) => rowsWithEditingCells(tableBodyRows, editingCells);
+const columnsWithEditingCellsComputed = (
+  { tableColumns, editingCells }: Getters,
+) => columnsWithEditingCells(tableColumns, editingCells);
+
 /* tslint:disable-next-line max-line-length*/
 const INLINE_CELL_EDITING_ERROR = 'The startEditAction property of the InlineCellEditing plugin is given an invalid value.';
 
 // tslint:disable-next-line: max-line-length
 const TableInlineCellEditingBase: React.SFC<TableInlineCellEditingProps> & {components: {cellComponent: string}} = (props) => {
   const { cellComponent: EditCell, startEditAction, selectTextOnEditStart } = props;
 
-  return(
+  return (
     <Plugin name="TableInlineCellEditing" dependencies={pluginDependencies}>
+      <Getter name="tableBodyRows" computed={rowsWithEditingCellsComputed} />
+      <Getter name="tableColumns" computed={columnsWithEditingCellsComputed} />
       <Template
         name="tableCell"
-        predicate={({ tableRow, tableColumn }: any) => (
-          tableRow.type === TABLE_DATA_TYPE && tableColumn.type === TABLE_DATA_TYPE
+        predicate={({ tableRow, tableColumn }: any) =>
+          tableRow.type === TABLE_DATA_TYPE &&
+          tableColumn.type === TABLE_DATA_TYPE
+        }
+      >
+        {(params: TableCellProps) => (
+          <TemplateConnector>
+            {({}, { startEditCells }) => {
+              const { tableRow, tableColumn } = params;
```

Comment:
it can be destructured in 1 call: `const { tableRow : { rowId, row }, tableColumn: { column } } = params;`